dist/initramfs/dracut/install

```diff
@@ -3,6 +3,7 @@
 inst_hook pre-mount 01 "$moddir/dattobd.sh"
 inst_dir /etc/datto/dla/mnt
 inst /sbin/blkid
+inst /sbin/blockdev
```

Comment:
@Conan-Kudo am I missing something here? This isn't getting added to the initramfs on  CentOS 7.